pipelineutilities/test/test_search_files.py

```diff
@@ -80,6 +83,7 @@ class TestSearchFiles(unittest.TestCase):
 
     def test_id_from_url(self):
         for url in temp_ids_example:
+            print("hi-----------")
```

Comment:
Probably didn't mean to leave this in.

Reply:
i would not be so sure.  I have a strong preference for friendly code.

Reply:
i would not be so sure.  I have a strong preference for friendly code.